packages/phoenix-ng/cypress/e2e/atlas.cy.ts

```diff
@@ -0,0 +1,78 @@
+describe('ATLAS', () => {
+  it('should be up and running', () => {
+    cy.visit('/atlas');
+  });
+
+  it('should display the ATLAS detector properly', () => {
```

Comment:
> Use [Cypress snapshots plugin](https://github.com/meinaart/cypress-plugin-snapshots) instead here.

Sure, I'll use it instead of checking for DOM elements.